### ISSUE TITLE
node 6 is unsupported now

### ISSUE THREAD

**Issue by ElvisChiang** [stephy/CalendarPicker#86]
72ecec0 actually removed node6 support 

```
yarn upgrade v0.27.5
[1/4] Resolving packages...
warning react-native-calendar-picker > react-native > xcode > node-uuid@1.4.7: Use uuid module instead
[2/4] Fetching packages...
error react-native-calendar-picker@5.10.0: The engine "node" is incompatible with this module. Expected version ">=7.6.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/upgrade for documentation about this command.
```

**Comment by peacechen** [comment]
This library shouldn't require any version-specific features in Node 7.x.  Recommend changing the package.json to:
```
 "engines" : {
    "node" : ">=4.0.0"
  },
```

I haven't tested Node 4.x, but seems like it would be better to support as many versions back as possible.

**Comment by derakhshanfar** [comment]
same issue for me.

**Comment by peacechen** [comment]
This has been fixed in the main branch but hasn't been published to npm.  You can point your package.json to this repo for the time being.

**Comment by derakhshanfar** [comment]
@peacechen thanks for reply but i removed it from package.json and when i want to rerun _react-native-git-upgrade_ command i get this error:

> fsevents@1.1.2: The platform "win32" is incompatible with this module.
> react-native-jalali-calendar-picker@0.2.6: The engine "node" is incompatible with this module. Expected version ">=7.6.0".
> Found incompatible module

**Comment by peacechen** [comment]
@derakhshanfar  Looks like you're using `react-native-jalali-calendar-picker`.  Might be a fork?

**Comment by derakhshanfar** [comment]
@peacechen i re-run npm install but it does not work. i get same error

> fsevents@1.1.2: The platform "win32" is incompatible with this module.
> react-native-jalali-calendar-picker@0.2.6: The engine "node" is incompatible with this module. Expected version ">=7.6.0".
> Found incompatible module

**Comment by trekare** [comment]
@peacechen is it possible to cut a release soon to fix the node >4 issue? Thanks, much appreciated.

**Comment by peacechen** [comment]
The current release (5.11.0) fixed the node version requirement.  Its package.json has
```
  "engines": {
    "node": ">=4.0.0"
  },
```

Please make sure you're using the current release in your package.json.
```
    "react-native-calendar-picker": "^5.11.0",
```

If that's specified, check that your package-lock.json points to the correct version too (or yarn if you use that).

**Comment by trekare** [comment]
@peacechen this is what npm sees for 5.11: https://unpkg.com/react-native-calendar-picker@5.11.0/package.json

Which seems in line with the fact that the nod 4.0 pr was merged on November 27th (https://github.com/stephy/CalendarPicker/pull/96)

![image](https://user-images.githubusercontent.com/5361430/35067842-1f5766e0-fb8a-11e7-817b-b4dec8638bf2.png)

and that the version bump was cut on November 16th

![image](https://user-images.githubusercontent.com/5361430/35067877-41bbb6e6-fb8a-11e7-8a11-0ac6ac1ccc99.png)

https://github.com/stephy/CalendarPicker/commit/92ae0b1989ec2bab6baa085b3db716700fca7342

**Comment by peacechen** [comment]
@trekare Thanks for noticing the discrepancy.  I got my wires crossed on the commits 😮 

I've published 5.12.0 which also includes a number of styling prop additions as well as conversion to Moment JS.  Users will need to add `moment` to the project's dependencies (see #95).
